reddit.js

```diff
@@ -120,6 +120,13 @@
     });
   };
 
+  reddit.user = function (username, where = "") {
```

Comment:
Please change this to `es5` compatible code by removing the default parameter. This is until we modernize the lib.